libc/cmake/modules/compiler_features/check_padding_on_unsigned_fixed_point.cpp

```diff
@@ -0,0 +1 @@
+#include "include/llvm-libc-macros/stdfix-macros.h"
```

Comment:
is this file supposed to have more? Maybe an `ifdef` to check the proper values are being defined? Notably, if you don't pass `-ffixed-point` to clang this will compile successfully, since `stdfix-macros.h` checks that before defining anything else.